build_tools/docker/bazel/gcr_update.sh

```diff
@@ -21,18 +21,19 @@ set -e
 # Ensure correct authorization.
 gcloud auth configure-docker
 
+# Determine which image tag to update. Updates :latest by default.
+if [[ $* == --update-prod ]]; then
+  TAG="prod"
+else
+  TAG="latest"
+fi
+echo "Updating $TAG"
+
 # Build and push the bazel image.
-docker build --tag gcr.io/iree-oss/bazel build_tools/docker/bazel/
-docker push gcr.io/iree-oss/bazel
+docker build --tag gcr.io/iree-oss/bazel:$TAG build_tools/docker/bazel/
```

Comment:
Put this in quotes and wrap the parameter expansion in braces. It's also the convention in our other scripts (because I made it so) to use a ? to cause errors on undefined parameters rather than silently using the empty string.
```suggestion
docker build --tag "gcr.io/iree-oss/bazel:${TAG?}" build_tools/docker/bazel/
```
https://google.github.io/styleguide/shellguide.html#s5.6-variable-expansion

Reply:
Done!

Reply:
(Aside from the ones that needed to be edited per your other comment)